src/components/SecondaryPanes/index.js

```diff
@@ -107,7 +108,28 @@ class SecondaryPanes extends Component<Props> {
         : L10N.getStr("breakpoints.disable")
     };
 
-    return <input {...inputProps} />;
+    if (!features.breakpointsDropdown) {
+      return <input {...inputProps} />;
+    }
+
+    return (
+      <div className="breakpoints-buttons">
+        {this.renderBreakpointsDropdown()}
+        <input {...inputProps} />
+      </div>
+    );
+  }
+
+  renderBreakpointsDropdown() {
+    const Panel = (
+      <ul>
+        <li>Pause on Next Statement</li>
```

Comment:
these should be i18n texts (for translation)

Reply:
https://github.com/devtools-html/debugger.html/blob/master/src/components/SecondaryPanes/CommandBar.js#L215